editor/sidebar/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -85,6 +85,9 @@ class HierarchicalTermSelector extends Component {
 	onAddTerm( event ) {
 		event.preventDefault();
 		const { formName, formParent } = this.state;
+		if ( formName === '' ) {
```

Comment:
I note similar errors of #2581 when term is nothing but whitespace. We should probably trim here before testing as empty.